aws/aws.go

```diff
@@ -41,6 +41,20 @@ func getRandomRegion() string {
 	return allRegions[randIndex]
 }
 
+func split(identifiers []string, limit int) [][]string {
```

Comment:
Please add a bunch of unit tests for this method to check boundary conditions: empty list, 1 item list, odd number of items, even number of items, small limit, large limit, etc.